tests/data/era5_dataset_test.py

```diff
@@ -0,0 +1,34 @@
+import os
+from climate_learn.data.modules import ERA5
+import pytest
+
+DATA_PATH = "/data0/datasets/weatherbench/data/weatherbench/"
+
```

Comment:
Just a style thing: can we get rid of the line between `DATA_PATH` and `GITHUB_ACTIONS`? Or does black forbid it?

Reply:
My bad. Black doesn't forbid it. I have removed the line.